internal/extproc/translator/openai_gcpvertexai.go

```diff
@@ -38,7 +43,20 @@ func (o *openAIToGCPVertexAITranslatorV1ChatCompletion) RequestBody(_ []byte, op
 		// Use modelName override if set.
 		modelName = o.modelNameOverride
 	}
-	pathSuffix := buildGCPModelPathSuffix(GCPModelPublisherGoogle, modelName, GCPMethodGenerateContent)
+
+	// Set streaming flag.
+	o.stream = openAIReq.Stream
+
+	// Choose the correct endpoint based on streaming.
+	var pathParams string
+	method := gcpMethodGenerateContent
+	if o.stream {
+		// For streaming requests, use the streamGenerateContent endpoint with SSE format.
+		method = gcpMethodStreamGenerateContent
+		pathParams = "alt=sse"
+	}
+
+	pathSuffix := buildGCPModelPathSuffix(gcpModelPublisherGoogle, modelName, method, pathParams)
```

Comment:
With this you don't need that checked i mentioned above.
```suggestion
	// Choose the correct endpoint based on streaming.
	var pathSuffix string
	if o.stream {
		// For streaming requests, use the streamGenerateContent endpoint with SSE format.
		pathSuffix = buildGCPModelPathSuffix(gcpModelPublisherGoogle, modelName, gcpMethodStreamGenerateContent, "alt=sse"
	} else {
		pathSuffix = buildGCPModelPathSuffix(gcpModelPublisherGoogle, modelName, gcpMethodGenerateContent)
	}
```